tailwind.config.js

```diff
@@ -226,6 +226,8 @@ module.exports = {
         sswRed: "#cc4141",
         sswDarkRed: "#8e2c2c",
         sswBlack: "#333333",
+        sswGray: "#797979",
+        sswDarkGray: "#666666",
```

Comment:
Is this an official SSW color? (i.e. from https://www.ssw.com.au/ssw/logo/ssw/) If not just get rid of it and use `gray-650` (duplicate value)

Reply:
I've got it from the TimePro, and I asked @jaydenalchin about it. I think we should use something like ssw-gray-dark for this instead, what do you think?